public/random/generic.html

```diff
@@ -82,83 +88,68 @@ <h2>First time</h2>
     </p>
 
     <br>
-
-    <p class="dialogue-line">
-        (Peeoo peeoo!)
-    </p>
-
-    <br>
-
-    <h2>Subsequent times</h2>
-
-    <p class="dialogue-line">
-        <span class="dialogue-head">
-            <span class="dialogue-name">Bird</span>
-        </span>
-        Piou piou!
-    </p>
-
-    <br>
-
-    <h4>ACT 1 & 2 only:</h4>
-
-    <p class="dialogue-line">
-        (Peeoo peeoo!)
-    </p>
+    
+    <ul>
+        <li>
+            <b>ACT 1 & 2 only:</b>
```

Comment:
personally the kind of thing i'd put in an indent but also i am the person here whose formatting is the most unique so

Reply:
actually yeah i do like the indented thing for one line as well, i wasn't sure if we'd agreed on one way to do it though lol

Reply:
yeah, the list looks pretty jarring to me personally, and also the bold looks too much like a name to me lol